pallets/nfts/src/lib.rs

```diff
@@ -223,7 +297,11 @@ pub mod pallet {
                 .clone()
                 .into_iter()
                 .for_each(|(account, details)| {
-                    drop(<Pallet<T> as NFTs>::create(&account, details))
+                    drop(<Pallet<T> as NFTs>::create(
+                        &account,
+                        details,
+                        Default::default(),
+                    ))
```

Comment:
Do we want to specify series during genesis?

Reply:
I guess :)

Currently instead of T::NFTSeriesId I used u32 since I was getting the following error:

> the trait `Deserialize<'_>` is not implemented for `<T as pallet::Config>::NFTSeriesId`

Reply:
Have you tried adding `MaybeSerializeDeserialize` to `NFTSeriesId` in `Config`?